src/main/java/weightScheme/SimpleWeight.java

```diff
@@ -14,9 +14,16 @@ public SimpleWeight(Weight weight) {
     @Override
     public double computeWeighted(double[] marks) {
         double exactWeightEarned = DoubleStream.of(marks)
+                .filter(mark -> mark >= 0)
                 .map(mark -> mark * weight.getWeightOfEachInstance())
                 .sum();
-        return exactWeightEarned / (marks.length * weight.getWeightOfEachInstance());
```

Comment:
You have changed the way marks are computed, which breaks compatibility with the rest of the program (outline and assessment's mark computation does not. Remember that if you'd like to add specific functionality for your own feature, NEVER edit existing functionality that was not authored by you unless you've discussed it with the original author. A better alternative is to create your own functions and use those as extensions, which leaves an opportunity to merge functionality later.

Reply:
That code is not mine. I pulled from [1-feature-course-with-outline](https://github.com/CSC207-2022F-UofT/course-project-justbob/tree/1-feature-course-with-outline). I am waiting for pull request #20 to be merged before pulling from there and creating another pull request. I will close this one.